CHANGELOG.md

```diff
@@ -455,3 +459,5 @@ Swift 3.0
 [#7722]: https://github.com/swiftlang/swift-package-manager/pull/7722
 [#7741]: https://github.com/swiftlang/swift-package-manager/pull/7741
 [#7813]: https://github.com/swiftlang/swift-package-manager/pull/7813
+[#8301]: https://github.com/swiftlang/swift-package-manager/pull/8301
+[#8314]: https://github.com/swiftlang/swift-package-manager/pull/8314
```

Comment:
suggestion: This link is not referenced in the update.  Can we either (1) remove it, or (2) use the reference?

Reply:
(ie. (2) is already true)

Reply:
Yeah, sorry I wasn't clear - 8314 is used by `strictMemorySafety` entry, it just wasn't declared here.